crates/router/src/connector/adyen/transformers.rs

```diff
@@ -2114,17 +2171,36 @@ impl<'a> TryFrom<(&api::PayLaterData, Option<api_enums::CountryAlpha2>)>
                 }
             }
             api_models::payments::PayLaterData::PayBrightRedirect { .. } => {
+                check_required_field(shopper_name, "shopper_name")?;
+                check_required_field(telephone_number, "telephone_number")?;
+                check_required_field(shopper_email, "email")?;
+                check_required_field(billing_address, "billing_address")?;
```

Comment:
Shouldn't we need to validate the inner params of shipping_address and billing_address?

Reply:
The inner fields requirements are not provided in the docs, the object is expected from Adyen so we are validating the object only.

Reply:
Adyen docs specifies either the complete Address Object is optional, if you are passing the objects then the certain fields inside the object are required so accordingly made that change